editor/src/core/workers/parser-printer/parser-printer.ts

```diff
@@ -2027,3 +2027,18 @@ export function lintAndParse(
     return parseFailure(null, null, null, lintResult)
   }
 }
+
+export function jsxAttributeToString(attribute: JSXAttribute): string {
+  switch (attribute.type) {
+    case 'ATTRIBUTE_VALUE':
+      if (typeof attribute.value === 'string') {
+        return attribute.value
+      } else {
+        return attribute.value.toString()
```

Comment:
`'[object Object]'`
Though I suspect if we wanted to use `JSON.stringify` here we'd have to be careful to handle all of the possible edge cases (type errors, deep objects).